src/main/GenomeTrack.js

```diff
@@ -27,7 +28,7 @@ var GenomeTrack = React.createClass({
     onRangeChange: React.PropTypes.func.isRequired,
   },
   render: function(): any {
-    return <div><canvas ref='canvas' /></div>;
+    return <canvas />;
   },
   componentDidMount: function() {
     var div = this.getDOMNode();
```

Comment:
Did not know you could do this, nice. (As an aside, are you using `div` here? (It's also `canvas` now (sorry for chiming in))).